definitions/vertical_scaling.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Vertical Scaling
+status: Completed
+category: concept
+---
+## Canary Deployment
+
+### What it is
+Vertical scaling refers to adding more resources (CPU/RAM/DISK) to your server (database or application server is still remains one) as on demand.
+
+Vertical Scaling is most commonly used in applications and products of middle-range as well as small and middle-sized companies. One of the most common examples of Virtual Scaling is to buy an expensive hardware and use it as a Virtual Machine hypervisor (VMWare ESX).
+
+Vertical Scaling usually means upgrade of server hardware. Some of the reasons to scale vertically includes increasing IOPS (Input / Ouput Operations), amplifying CPU/RAM capacity, as well as disk capacit
+### Problem it Addresses
+Scalability is essential in that it contributes to competitiveness, efficiency, reputation and quality. Vertical scaling is on one of the two kinds (Vertical,Horizontal) of scalability.
```

Comment:
We already have scalability on it's own as a property. Worth linking to it here. Stick to the problem vertical scaling addresses, namely: 

The need to have an instance of an application handle a higher load of transactions. 

You can also reference horizontal scaling scaling here as another way to add more throughput to an app.